$NetBSD: patch-ak,v 1.1 2007/10/09 11:09:23 rillig Exp $

Warning: typos can greatly reduce the readability of your code (see
patch-aa).

--- Xpm.h.orig	1996-11-17 08:13:02.000000000 +0100
+++ Xpm.h	2007-10-09 12:52:11.000000000 +0200
@@ -8,13 +8,13 @@ typedef struct _PixmapNode {
         Pixmap pixmap;
 	Pixmap mask;
         int usage;
-        struct PixmapNode *next;
+        struct _PixmapNode *next;
 } PixmapNode;
 
 typedef struct _pixmaplist{
         int n_entries;
-        struct PixmapNode *first;
-        struct PixmapNode *last;
+        struct _PixmapNode *first;
+        struct _PixmapNode *last;
 } pixmaplist;
 
 pixmaplist PixList;
